package.json

```diff
@@ -1,31 +1,45 @@
 {
   "name": "formidable-react-component-boilerplate",
   "version": "0.0.1",
-  "description": "",
-  "main": "index.js",
-  "dependencies": {
+  "description": "react component boilerplate",
+  "main": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/FormidableLabs/formidable-react-component-boilerplate.git"
   },
-  "peerDependencies": {
-    "react": "0.13.x"
+  "author": "Lauren Eastridge",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/FormidableLabs/formidable-react-component-boilerplate/issues"
+  },
+  "homepage": "https://github.com/FormidableLabs/formidable-react-component-boilerplate",
+  "scripts": {
+    "postinstall": "npm run clean && npm run lib",
```

Comment:
I'm going to take a little time on Monday to standardize these things. We should be wary of doing a lot on `postinstall`... and I'd like to hone us down to a smaller number of convention-based commands.